lms/djangoapps/bulk_enroll/views.py

```diff
@@ -72,9 +82,23 @@ def post(self, request):
                 'action': serializer.data.get('action'),
                 'courses': {}
             }
-            for course in serializer.data.get('courses'):
-                response = students_update_enrollment(self.request, course_id=course)
-                response_dict['courses'][course] = json.loads(response.content)
+            for course_id, cohort_name in itertools.izip_longest(serializer.data.get('courses'),
```

Comment:
This view should raise an error if the number of items in `courses` and `cohorts` are not equal.

Reply:
Catered for this in the seralizer's validation.